yet-another-docker-plugin/src/main/resources/com/github/kostyasha/yad/launcher/DockerComputerJNLPLauncher/help-jnlpOpts.html

```diff
@@ -0,0 +1,3 @@
+<dir>
+    Java Web Start agent options (JNLP). <a href="https://wiki.jenkins-ci.org/display/JENKINS/Distributed+builds#Distributedbuilds-LaunchslaveagentviaJavaWebStart">For details</a> or use java -jar slave.jar -help
```

Comment:
Should be enough, but better check `java -jar slave.jar --help` for all available options.